### ISSUE TITLE
Example for Extended OPF Formulation

### ISSUE THREAD

**Issue by MuriloReolonScuzziato** [MATPOWER/matpower#237]
Is there some example that uses Extended OPF Formulation? I would like to know how to change the objective function and/or add variables and constraints to the opf problem. Having an example considering the minimization of the total branch losses, for instance, for any of the cases in the data folder would be very useful.

**Comment by rdzman** [comment]
Yes. See chapter 7, especially section 7.6 in the [User's Manual](https://matpower.org/docs/MATPOWER-manual-8.0.pdf) for some examples. The zonal reserves implementation (see [`toggle_reserves.m`](https://github.com/MATPOWER/matpower/blob/master/lib/toggle_reserves.m)) in particular is pretty complete and described in detail in that chapter.

If all you need is to minimize total branch losses, see [FAQ #1](https://matpower.org/doc-original/faq/#lossminimization).
